drivers/hmis/app/graphql/types/hmis_schema/enums/hud.rb

```diff
@@ -614,4 +614,22 @@ class AdHocYesNo < Types::BaseEnum
     graphql_name 'AdHocYesNo'
     hud_enum ::HudLists.ad_hoc_yes_no_map
   end
+
+  class RentalSubsidyType < Types::BaseEnum
```

Comment:
I removed this for now, and commented out the RentalSubsidyType fields, since they are new in 2024. This enum file is generated based off hud_lists.json, so once we create the [2024 version](https://www.pivotaltracker.com/story/show/185275463) and switch over to it, we'll have the enum and can add back the fields.

Reply:
Sounds good. It was listed in the HUD spec so I figured it should be included, but that makes sense.